Makefile

```diff
@@ -407,6 +390,18 @@ internal-clean: $$(KIND)
 internal-distclean: $$(KIND)
 	@rm -rf ./_build/$(KIND)
 
+internal-ct: $$(KIND)
+internal-ct: $$(SYSCONFIG)
+internal-ct: $$(AETERNITY_TESTCONFIG_DB_BACKEND)
+internal-ct: internal-build
+	@NODE_PROCESSES="$$(ps -fea | grep bin/aeternity | grep -v grep)"; \
+	if [ $$(printf "%b" "$${NODE_PROCESSES}" | wc -l) -gt 0 ] ; then \
+		(printf "%b\n%b\n" "Failed testing: another node is already running" "$${NODE_PROCESSES}" >&2; exit 1);\
+	else \
+		AETERNITY_TESTCONFIG_DB_BACKEND=$(AETERNITY_TESTCONFIG_DB_BACKEND) \
```

Comment:
I typed this as `AETERNITY_TESTCONFIG_DB_BACKEN=$(AETERNITY_TESTCONFIG_DB_BACKEND)` and I got:
```
*** CT Error Notification 2019-04-02 15:44:25.166 ***
aesc_fsm_SUITE:init_per_suite failed
Reason: {test_case_failed,The mandatory environment variable is not set: AETERNITY_TESTCONFIG_DB_BACKEND}
```